DeUrgenta/Features/Menu/View/MenuView.swift

```diff
@@ -0,0 +1,70 @@
+import SwiftUI
+
+protocol MenuViewDelegate {
+    func menuViewShouldRedirect(from view: MenuView, withItem item: MenuOption)
+    func menuViewDidTapDonateButton(_ view: MenuView)
+}
+
+struct MenuView: View {
+    @ObservedObject var viewModel: MenuViewModel
+    var delegate: MenuViewDelegate
+
+    var body: some View {
+        VStack(spacing: 0) {
+            ScrollView {
+                HStack {
+                    Image("logo")
+                        .resizable()
+                        .frame(width: 100, height: 100)
+
+                    Spacer()
+                }
+                .padding()
+
+                ForEach(viewModel.options, id: \.id) { option in
+                    VStack {
+                        Button(action: {
+                            delegate.menuViewShouldRedirect(from: self, withItem: option)
+                        }, label: {
+                            VStack(alignment: .leading) {
+                                Text(option.name)
+                                    .font(.custom("IBMPlexSans-Bold", size: 16))
+                                    .foregroundColor(.darkText)
+                                    .padding()
+
+                                Divider()
+                            }
+                        })
+                    }
+                }
+
+                Spacer()
+
+                VStack(alignment: .leading) {
+                    Text(AppStrings.MenuView.headerIconLabel.localized())
+                        .font(.custom("IBMPlexSans-Regular", size: 14))
+                        .foregroundColor(Color.gray)
+
+                    Image("code4romania-icon")
+
+                    Button(action: {
+                        delegate.menuViewDidTapDonateButton(self)
+                    }, label: {
+                        HStack {
+                            Text(AppStrings.MenuView.donateButtonText.localized())
+                                .font(.custom("IBMPlexSans-Bold", size: 16))
+                                .foregroundColor(.secondary)
+                                .padding(.vertical)
+                        }
+                        .frame(maxWidth: .infinity, maxHeight: 50)
+                        .background(Color.green)
+                        .cornerRadius(6)
+                    })
+                    .padding(.vertical, 20)
+                }
+                .padding(.top, 80)
+                .padding(.horizontal)
+            }
+        }
```

Comment:
The view is pretty hard to read, maybe we can extract some subview components? Not necessarily in a separate file, but as `vars` in the view.